src/pages/documentation/mobile-core/lifecycle/metrics.md

```diff
@@ -1,101 +0,0 @@
-# Lifecycle metrics
```

Comment:
Maybe we can preserve this page here if we consolidate the information from index.md also - see also https://github.com/AdobeDocs/acp-mobile-sdkdocs/pull/7/files#r1028360645